docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -318,7 +318,7 @@ For information on YB-Master load balancing, see [Data placement and load balanc
 
 For load balancing commands in `yb-admin`, see [Rebalancing commands (yb-admin)](../../../admin/yb-admin/#rebalancing-commands).
 
-For information on internal load balancing to power geo-distributed applications, see [Yugabyte JDBC Driver](../../../integrations/jdbc-driver).
+For information on internal load balancing to power geo-distributed applications, see [YugabyteDB JDBC Driver](../../../reference/drivers/java/yugabyte-jdbc-reference/).
```

Comment:
Correct link? Section is about internal data load balancing pretty sure; placement of tablets/shards across the nodes, ie not connection load balancing. Not sure it's all that relevant to drivers, or at least only peripherally

Reply:
I agree , I added this when we introduced the JDBC smart driver. It is not relevant to the information on that page in particular. Will remove it. Thanks.